controller/pokemon.go

```diff
@@ -0,0 +1,29 @@
+package controller
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/MoraAlex/academy-go-q32021/services"
+	"github.com/gorilla/mux"
+)
+
+func GetAllPokemons(w http.ResponseWriter, r *http.Request) {
```

Comment:
By convention, all the exported stuff (function, variable, constant, interface, struct, etc) should have a comment with a description:
```go
// GetAllPokemons - Description here...
func GetAllPokemons(w http.ResponseWriter, r *http.Request) {
```